parser/ast/dml.go

```diff
@@ -2573,6 +2573,13 @@ type Limit struct {
 	Offset ExprNode
 }
 
+type LimitSimple struct {
+	node
+
+	Count  int64
+	Offset int64
+}
+
```

Comment:
Strange that `LimitSimple` is a `node`, but there is no restoring function for it. It should be put closely to admin stmt, rather than select.